### ISSUE TITLE
Is there a Changelog or Release Notes

### ISSUE THREAD

**Issue by sureshjoshi** [OfficeDev/office-js-docs#1510]
As the question is titled, I was just wondering if there is an official change log or release notes for each update of Office JS?

When I periodically update, and suddenly a lot of my Types no longer function, it's hard to figure out what changed, what the migration strategy is, etc - without having a set of notes as to what changed.

I resort to needing to try to read through the merge commits from the DefinitelyTyped repo, which is unwieldy.

**Comment by dmahugh** [comment]
Hi Suresh, there is a page listing what's new in each requirement set on the Office Dev Center. For example, here are the links to the latest documentation for [Excel](https://docs.microsoft.com/en-us/office/dev/add-ins/reference/requirement-sets/excel-api-requirement-sets), [Word](https://docs.microsoft.com/en-us/office/dev/add-ins/reference/requirement-sets/word-api-requirement-sets), and [Outlook](https://docs.microsoft.com/en-us/office/dev/add-ins/reference/requirement-sets/outlook-api-requirement-sets). You can see there which types have been changed or added in the latest requirement set.

**Comment by sureshjoshi** [comment]
Hi @dmahugh - thanks for your reply and the links.

But to be clear, the answer to my question is "No" correct? 

The requirement set is definitely useful at a high level and it's great from an API listing point of view, but it's not really a changelog or release notes (which I would expect on each release to provide added features, removed features, changed features, deprecated features, removed features, bug fixes, security fixes, migration strategy, etc...). 

Basically just a lot more granularity so I can answer questions like "Why did upgrading from "@types/office-js": "^0.0.63", to "@types/office-js": "^0.0.104" cause my application to no longer compile?" - I know this is a DefinitelyTyped example, but the point is, this did happen to me and I was reading thousands of lines of source code for hours to figure out what happened.

**Comment by dmahugh** [comment]
Yeah, unfortunately we don't have that level of documentation detail for each requirement set.  If you're working with Outlook, you'll see that they provide a bit more detail on their change log - for example, see the [Requirement set 1.7 Change log](https://docs.microsoft.com/en-us/office/dev/add-ins/reference/objectmodel/requirement-set-1.7/outlook-requirement-set-1.7).

This is good feedback, which I'll pass on to the team.

**Comment by sureshjoshi** [comment]
Thanks @dmahugh I would appreciate that. I have a few colleagues who are in the same boat, and we're all kinda struggling.
